autoendpoint/src/routers/fcm/settings.rs

```diff
@@ -11,6 +11,24 @@ pub struct FcmSettings {
     pub min_ttl: usize,
     /// A JSON dict of `FcmCredential`s. This must be a `String` because
     /// environment variables cannot encode a `HashMap<String, FcmCredential>`
+    /// This contains both GCM and FCM credentials.
```

Comment:
We should really mention how they're distinguished here as well (existence of "{" vs base64)) with the examples below reflecting that, but I'm fine deferring this to a separate issue though.

Reply:
I thought I did that at line 27, where I have credentials for both, or do you mean something else?

Reply:
... and I found out that endpoint isn't optionally treating the credential value as a path identifier, like what the python version was doing.